### ISSUE TITLE
Failure to claim the second freebies

### ISSUE THREAD

**Issue by YogaSakti** [Revadike/epicgames-freebies-claimer#107]
### Expected Behavior

Successfully claimed 2 items

### Actual Behavior
A failure occurred when claiming the 2nd game

![image](https://user-images.githubusercontent.com/24309806/122593771-567f8d80-d090-11eb-8976-13185e7d858f.png)

I've tried running it again and got the same result

![image](https://user-images.githubusercontent.com/24309806/122595116-2507c180-d092-11eb-9e42-8f3120e264ce.png)

### Steps to Reproduce the Problem

  1. Add 4 Account in device_auth 
  2. Run as in instructions (No configuration changed)

### Specifications

  - `epicgames-freebies-claimer` Version: 1.50
  - Platform / OS: Ubuntu 18.04.5 LTS
  - Node Version: v14.17.0

**Comment by ItsEx0tical** [comment]
Same here.

**Comment by xmsama** [comment]
+1

**Comment by antoinerzad** [comment]
same

**Comment by zaevi** [comment]
Same here.
I claim the game manually and try running again, and it still occur this failure.

**Comment by lhllhx** [comment]
same

**Comment by Revadike** [comment]
Writing _same_ does not help. It would help much more if someone would look into this more closely and share some info about how this happens, or even a PR. Thank you.

**Comment by zaevi** [comment]
more details:
```
Error: There was a problem processing your request.  Please try again later.
    at Launcher.purchaseOrderPreview (.\\node_modules\\epicgames-client\\src\\Client\\index.js:628:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Launcher.purchase (.\\node_modules\\epicgames-client\\src\\Client\\index.js:671:19)
    at async .\\claimer.js:60:37
```

Seems `freeGamesPromotions` get wrong offer. `Overcooked! 2` have two editions:
 - Gourmet Edition, not published yet, id: `fc17f11ea7f64f0bad10c9efbdc9b9ee`, the method returned this offer
 - basic edition, free in this week, id: `0fd170f41c284239a6a6e0be2df3b8e5`

**Comment by Revadike** [comment]
Nice catch!
